### PR TITLE
Target c++17

### PR DESCRIPTION
**Pull request by jkotalik**
We want a few features in c++17 (like std::filesystem) for future PRs. Setting the version of c++ to 2017 in all vcxproj files we use.

### CODE REVIEW

File: src/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj [hunk 1]
```diff
@@ -97,6 +97,7 @@
       <CompileAs>CompileAsCpp</CompileAs>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
+      <LanguageStandard>stdcpp17</LanguageStandard>
```
Comment (pan-wang) [inline src/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj]: is there any size change for the output assemblies? have you tested it on Win7 box?

Comment (jkotalik) [reply src/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj]: Size is the same. I don't see why I would need to test this on a win7 box, but sure.

Comment (jkotalik) [reply src/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj]: Looks fine on win7.

**Comment by jkotalik** [comment]
@pan-wang this good to go?

**Pull request merged**